recipe/build.sh

```diff
@@ -23,11 +22,11 @@ export CPPFLAGS="${CPPFLAGS} -DSIOCGSKNS=0x894C"
             --disable-chfn-chsh  \
             --disable-login      \
             --disable-nologin    \
-            --disable-uuidd      \
             --disable-su         \
             --disable-setpriv    \
             --disable-runuser    \
             --disable-static     \
+            --disable-libuuid     \
```

Comment:
we can probably enable this when 
https://github.com/conda-forge/libuuid-feedstock/pull/27

is merged

Reply:
I've managed to enable this when https://github.com/conda-forge/libuuid-feedstock/pull/27 is merged in.